.github/actions/publish-image/action.yml

```diff
@@ -112,12 +112,20 @@ runs:
 
         # Generate the SBOM
         syft scan \
-          --output cyclonedx-json=sbom.json \
-          --select-catalogers "-cargo-auditable-binary-cataloger" \
+          --output cyclonedx-json@1.5=sbom_raw.json \
+          --select-catalogers "-cargo-auditable-binary-cataloger,+sbom-cataloger" \
           --scope all-layers \
           --source-name "$IMAGE_REPOSITORY" \
           --source-version "$IMAGE_MANIFEST_TAG" "${IMAGE_REPO_DIGEST}"
 
+        # Merge SBOM components using https://github.com/stackabletech/mergebom
+        curl -L -o mergebom https://repo.stackable.tech/repository/packages/mergebom/stable-$(arch)
```

Comment:
Can you add the `--fail` flag to all `curl` stataments please?

Reply:
```suggestion
        curl --fail -L -o mergebom https://repo.stackable.tech/repository/packages/mergebom/stable-$(arch)
```

Reply:
Both done, thanks for the suggestion @NickLarsenNZ

Reply:
Well I'll be... `arch` is part of coreutils (as an alias to `uname -m`). Wonder why it didn't work in Nix?

https://en.wikipedia.org/wiki/List_of_GNU_Core_Utilities_commands#Shell_Utilities